frontend/src/features/SideWindow/MissionForm/ActionForm/AirControlForm.tsx

```diff
@@ -0,0 +1,82 @@
+import { FormikDatePicker, FormikEffect, Icon, MultiZoneEditor } from '@mtes-mct/monitor-ui'
+import { Formik } from 'formik'
+import { noop } from 'lodash'
+import { useMemo } from 'react'
+import * as Yup from 'yup'
+
+import { FormikMultiInfractionPicker } from './shared/FormikMultiInfractionPicker'
+import { getTitleDateFromUtcStringDate } from './shared/utils'
+import { VesselField } from './shared/VesselField'
+import { useNewWindow } from '../../../../ui/NewWindow'
+import { FormBody } from '../FormBody'
+import { FormHead } from '../FormHead'
+
+import type { MissionActionFormValues } from '../types'
+import type { Promisable } from 'type-fest'
+
+const MissionActionInfractionSchema = Yup.object().shape({
+  comments: Yup.string().required(),
```

Comment:
C'est required ?

Reply:
Yes according to existing types: https://github.com/MTES-MCT/monitorfish/blob/master/frontend/src/domain/types/missionAction.ts#L44

Reply:
But a string type can be empty right ? Here also ?

Reply:
Yup can allow or not empty strings while requiring the prop to exist, I just have to chain other validators. What's the usage? Do we want to allow empty observation or not?